recipe/meta.yaml

```diff
@@ -11,10 +11,12 @@ source:
     - cblas.diff
 
 build:
-  number: 2
-  skip: true  # [py27]
+  number: 3
+  skip: true  # [py>39]
```

Comment:
Does this skip python 3.10?

Reply:
Indeed - python 3.10 is only supported by numpy since 1.21.3

Reply:
Ahh, I see. This is numpy 1.19.5 only